inlong-tubemq/tubemq-core/src/main/java/org/apache/inlong/tubemq/corerpc/netty/NettyClientFactory.java

```diff
@@ -235,15 +238,13 @@ private Client createClient(final NodeAddrInfo addressInfo,
                 conf.getInt(RpcConstants.NETTY_WRITE_HIGH_MARK, -1);
         int nettyWriteLowMark =
                 conf.getInt(RpcConstants.NETTY_WRITE_LOW_MARK, -1);
-        if (nettyWriteHighMark > 0) {
-            clientBootstrap.option(ChannelOption.WRITE_BUFFER_HIGH_WATER_MARK, nettyWriteHighMark);
-        }
-        if (nettyWriteLowMark > 0) {
-            clientBootstrap.option(ChannelOption.WRITE_BUFFER_LOW_WATER_MARK, nettyWriteLowMark);
+        if (nettyWriteHighMark > 0 && nettyWriteLowMark > 0) {
```

Comment:
What if the user only needs to modify nettyWriteHighMark or nettyWriteLowMark?

Reply:
@haibo-duan

Reply:
In the previous code,if nettyWriteHighMark not set ,the default value is 64 * 1024. if nettyWriteLowMark not set ,the default value is 32 * 1024. I have modified the code,  please help me review it. thanks!